### DISCUSSION TITLE
Database Driver - Progress

### DISCUSSION THREAD

**Discussion by neklein** [ctwillie/expo-server-sdk-php#25] [Ideas]
Hi @ctwillie - any word on the database driver progress?

I am creating my own in a fork, though I'm not sure it will merge back in smoothly - I'll see what I can do as far as making it generally useful, unless you have something close to done.

**Comment by ctwillie** [comment]
As of now, there are no plans to add the database driver. I initially thought it was a good idea, but now think that is outside the scope of this small package. Projects should be responsible for storing subscribed tokens and simply use this package to send the notification.
